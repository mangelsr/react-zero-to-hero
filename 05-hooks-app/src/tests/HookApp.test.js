import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';

import { HooksApp } from '../HooksApp';


describe('HookApp component test', () => {
    test('should match snapshot', () => {
        const wrapper = shallow(<HooksApp />);
        expect(wrapper).toMatchSnapshot();
    }); 
});
 