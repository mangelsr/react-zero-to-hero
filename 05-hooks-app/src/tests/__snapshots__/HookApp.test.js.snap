// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp component test should match snapshot 1`] = `
<div>
  <h1>
    Hola mundo!
  </h1>
</div>
`;
